src/js/client/templateManager.js

```diff
@@ -14,12 +14,14 @@
         gradeNames: ["fluid.component"],
         components: {
             renderer: {
-                type: "gpii.handlebars.renderer.serverAware"
+                // TODO sgithens This needs to be updated depending on the app from serverAware to
+                // serverMessageAware dynamically.
+                type: "gpii.handlebars.renderer.serverMessageAware"
```

Comment:
You could argue that we could put comments next to any type when there are variants, but that puts the burden on implementers to know which code to read and to read it.  If this switch is required often enough, maybe a mix-in grade that does this would help?  i.e. a templateManager.serverMessageAware grade that is also mentioned in the docs and used in tests?